src/server.ts

```diff
@@ -150,6 +152,17 @@ function browsersListParser(data: string): IBrowsersList {
 		.map((line: string) => line.trim());
 }
 
+function getDocumentPath(document: string): string {
+	return document
+		.replace(workspaceFolder, '')
+		.replace(/[^\\]*$/g, '');
```

Comment:
If I understand correctly what's going on here, it will not work on Unix-like systems, because path's in this systems has a `/` separator. Can you tell me why you need to do this?

Reply:
You're totally right. I was in a bit of a hurry to finish up...
So, the idea was to make the extension able to process different paths  based on different `browserslist` config. I also wanted to log which path is processed with which browser scope.
Of course, I did not want to log a "path-browser scope" combo as many times as many files there are on the given path, so I "save" each string in the global `loggedBrowserScope` variable.

The outcome is the following:
```
[Info  - 5:32:26 PM] The browser scope for test/ is "ie 6-8"
[Info  - 5:32:47 PM] The browser scope for test/sub-test/ is "ie 8"
```

Reply:
[ ] Replace `.replace(workspaceFolder, '')` with `path.relative()`
[ ] Replace windows specific RegExp with a Unix-friendly version